zerver/lib/message.py

```diff
@@ -574,8 +575,8 @@ def render_markdown(message: Message,
     if realm is None:
         realm = message.get_realm()
 
-    if realm_alert_words is None:
-        realm_alert_words = dict()
+    if realm_alert_words_automaton is None:
+        realm_alert_words_automaton = ahocorasick.Automaton()
```

Comment:
Same comment on None here as above; is the right behavior to just pass `None` in this case?  If so, we don't need this block.

Reply:
`do_render_markdown` doens't take `None` for `realm_alert_words_automaton ` or for `realm_alert_words ` earlier. So this block will be required unless we make changes to `do_render_markdown`